dendrite-config.yaml

```diff
@@ -80,6 +80,35 @@ global:
     # The roomname to be used when creating messages
     room_name: "Server Alerts"
 
+  # Consent tracking configuration
+  user_consent:
+    # If the user consent tracking is enabled or not
+    enabled: false
+    # The base URL this homeserver will serve clients on, e.g. https://matrix.org
+    base_url: http://localhost
```

Comment:
I wonder if this should be elsewhere given it's surely specified in other places? Well-known?

Reply:
I feel it's better to have this here, so it's clear that it belongs to the consent tracking, as an alternative I can move the `base_url` to global